### DISCUSSION TITLE
Decrease indent removes list

### DISCUSSION THREAD

**Discussion by lkxed** [tinymce/tinymce#9260] [General]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](fiddle.tiny.cloud) or similar.
1.  Just open [http://fiddle.tinymce.com](http://fiddle.tinymce.com) with default settings
2. Type any word
3. Make it numbering list
4. Decrease indent

### ✔️ Expected result
*What is the expected result of the above steps?*

The list is preserved.

### ❌ Actual result
*What is the actual result of the above steps?*

The list is removed.

## 📃 Other details
- Browser: Edge, Chrome, Firefox
- OS: Windows
- First affected version: …
- Worked in version: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by lkxed** [comment]
If the "remove" is somehow expected, how do I change the default behavior through configuration? Appreciate any help.

**Comment by lostkeys** [comment]
Hi @lkxed, Since this behavior is by design, I converted your issue to a discussion. I looked at some editors and it is really an opinionated decision and there exist no best practices. Some editors don't allow you to outdent a root-level list, but some do convert it back to a paragraph.

What are your (and others reading this) business/ux arguments for us changing this?


— Fredrik, Product Manager

**Reply by lkxed** [reply]
Hi @lostkeys, thanks for reply. 

> Some editors don't allow you to outdent a root-level list

That's exactly what I want. Because IMO outdent should be opposite operation of indent. If increase indent doesn't create a list, then why decrease indent should remove a list? Also, from a user perspective, decrease indent does what it doesn't say to do. It says to decrease indent but actually it could remove a list too, that's unexpected to user. 

That being said, I still need a workaround to disable/invalidate outdent for a root-level list for my urgent needs now. Do you know if there is any way I can achieve that?

Again thanks for your support.

**Reply by lostkeys** [reply]
Hi @lkxed. Unfortunately there exist no supported config option to change this behavior.

**Reply by lkxed** [reply]
Thanks anyway @lostkeys. I managed to achieve what I want by adding a custom toolbar button and calling `editor.execCommand("outdent")` conditionally during `onAction`.
